app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/task_list.scala.html

```diff
@@ -34,6 +39,39 @@
 @propertyAddressState = @{ draftReturn.propertyAddress.fold[TaskListStatus](TaskListStatus.ToDo)(_ => TaskListStatus.Complete) }
 @disposalDetailsState = @{ draftReturn.disposalDetailsAnswers.fold[TaskListStatus](TaskListStatus.ToDo)(_.fold(_ => TaskListStatus.InProgress,_ => TaskListStatus.Complete)) }
 @acquistionDetailsState = @{ draftReturn.acquisitionDetailsAnswers.fold[TaskListStatus](TaskListStatus.ToDo)(_.fold(_ => TaskListStatus.InProgress,_ => TaskListStatus.Complete)) }
+
+@initialGainOrLossState = @{
+    def canUseService = {
+        val residentialBeforeDate = LocalDateUtils.dateFromString("1/04/2015")
```

Comment:
you can use `LocalDate.of(2015,4,1)` instead which is a bit safer

Reply:
Done